cookbooks/cdo-apps/metadata.rb

```diff
@@ -8,6 +8,7 @@
 
 depends 'apt'
 depends 'build-essential'
+depends 'yarn'
```

Comment:
I'm surprised we weren't already depending on this.

Reply:
That's a dependency on the yarn cookbook, not yarn itself - makes it more convenient to install stuff with yarn from chef cookbooks. Afaik all our other stuff does yarn through rake tasks and such, not chef.